frontend/src/__tests__/cypress/cypress/e2e/pipelines/PipelineRuns.cy.ts

```diff
@@ -493,6 +501,178 @@ describe('Pipeline runs', () => {
+        // it('filter by started', () => {
+        //   // Verify initial run rows exist
+        //   archivedRunsTable.findRows().should('have.length', 2);
+
+        //   // Select the "Started" filter, select a value to filter by
+        //   pipelineRunsGlobal
+        //     .findArchivedRunsToolbar()
+        //     .within(() => pipelineRunsGlobal.selectFilterByName('Started'));
+
+        //   // Mock runs (filtered by start date), type a start date
+        //   archivedRunsTable.mockGetArchivedRuns(
+        //     mockArchivedRuns.filter((mockRun) => mockRun.created_at.includes('2024-02-05')),
+        //     1,
+        //   );
+        //   pipelineRunsGlobal
+        //     .findArchivedRunsToolbar()
+        //     .within(() => pipelineRunFilterBar.findStartDateInput().type('2024-02-05'));
+
+        //   // Verify only rows with selected start date exist
+        //   archivedRunsTable.findRows().should('have.length', 1);
+        //   archivedRunsTable.getRowByName('Test archived run 1').find();
+
+        //   // Mock runs with a cleared filter before updating again
+        //   archivedRunsTable.mockGetRuns(mockArchivedRuns, [], 1);
+        //   pipelineRunsGlobal
+        //     .findArchivedRunsToolbar()
+        //     .within(() => pipelineRunFilterBar.findStartDateInput().clear());
+
+        //   // Mock runs with a start date not associated with those runs
+        //   archivedRunsTable.mockGetArchivedRuns(
+        //     mockArchivedRuns.filter((mockRun) => mockRun.created_at.includes('2024-02-15')),
+        //     1,
+        //   );
+        //   pipelineRunsGlobal
+        //     .findArchivedRunsToolbar()
+        //     .within(() => pipelineRunFilterBar.findStartDateInput().type('2024-02-15'));
+
+        //   // Verify no results were found
+        //   archivedRunsTable.findEmptyResults().should('exist');
+        // });
+
+        it('filter by status', () => {
+          // Verify initial run rows exist
+          archivedRunsTable.findRows().should('have.length', 2);
+
+          // Select the "Status" filter
+          pipelineRunsGlobal
+            .findArchivedRunsToolbar()
+            .within(() => pipelineRunsGlobal.selectFilterByName('Status'));
+
+          // Mock runs (filtered by a status of 'SUCCEEDED')
+          archivedRunsTable.mockGetArchivedRuns(
+            mockArchivedRuns.filter((mockRun) => mockRun.state === RuntimeStateKF.SUCCEEDED),
+            1,
+          );
+          // Select a filter value of 'SUCCEEDED'
+          pipelineRunsGlobal
+            .findArchivedRunsToolbar()
+            .within(() =>
+              pipelineRunFilterBar.selectStatusByName(runtimeStateLabels[RuntimeStateKF.SUCCEEDED]),
+            );
+
+          // Verify only rows with the selected status exist
+          archivedRunsTable.findRows().should('have.length', 2);
+          archivedRunsTable.getRowByName('Test archived run 1').find();
```

Comment:
Should works here instead of find()

Reply:
`getRowByName(...).find().should('exist')` is what we should be doing. The `should('exist')` isn't actually needed but it it explicitly shows the _intent_ of the code.

That being said, please find all instances where `getRowByName(...)` is called without any assertions or chaining afterwards because simply calling a `get` test function does not perform any assertions or selections, it just returns a new object.